src/veh_interact.cpp

```diff
@@ -2092,6 +2090,15 @@ void veh_interact::display_details( const vpart_info *part )
         }
     }
 
+    if( part->has_flag( VPFLAG_WHEEL ) ) {
+        std::string label;
```

Comment:
this variable is excessive in this case, just put proper strings instead of ```label.c_str()``` below

Reply:
> just put proper strings instead of label.c_str() below

Proper strings?

Reply:
You got it right.